# Input
float64[] v_body  # chiều dài 3: [v_x, v_y, omega_z]
---
# Output
float64[] omegas   # chiều dài 3: [omega1, omega2, omega3], đơn vị rad/s
